workflow/setup_rocoto.py

```diff
@@ -121,18 +121,25 @@
                     break
             if not found:
                 outfile.write(line)
-    #
+    # ~~~~~~~~~~~~
 
 setup_xml(HOMErrfs, expdir)
 
-if os.getenv('YAML_GEN_METHOD', '1') == '1':
-    srcdir = f'{HOMErrfs}/workflow/ush/qrocoto'
-    dstdir = f'{expdir}/qrocoto'
-    if os.path.exists(dstdir):
-        shutil.rmtree(dstdir)
-    shutil.copytree(srcdir, dstdir)
+# copy qrocoto utilities to expdir/qrocoto
+srcdir = f'{HOMErrfs}/workflow/ush/qrocoto'
+dstdir = f'{expdir}/qrocoto'
+if os.path.exists(dstdir):
+    shutil.rmtree(dstdir)
+shutil.copytree(srcdir, dstdir)
 
-elif os.getenv('YAML_GEN_METHOD', '1') == '2':
+if os.getenv("DO_JEDI", 'false').upper() == "TRUE":
+    print('\nIf doing satellite radiance data assimilation, run the following commands to prepare the initial satbias files:')
+    print(f'    cd  {expdir}')
+    print(f'    source qrocoto/load_qrocoto.sh')
+    print(f'    prep_satbias.sh YYYYMMDDHH [satbias_path]')
+    print(f'check https://github.com/NOAA-EMC/rrfs-workflow/wiki for more details')
+
+if os.getenv('YAML_GEN_METHOD', '1') == '2':
```

Comment:
This looks like if you use `YAML_GEN_METHOD=2` you'll get both sets of tools.

Reply:
No matter what method you use, we will need the `prep_stabias` utility to prepare the initial stabias files.
Check this
https://github.com/NOAA-EMC/rrfs-workflow/wiki/Satellite-Radiance-DA-prepare-initial-satbias-files-and-others
for how to prepare the initial satbias files.

Reply:
Why is `prep_satbias.sh` reliant on the qrocoto tools? Shouldn't it be its own stand alone tool?

Reply:
I had thought that since there are already multiple additions in your expdir, it may be fine to add a `qrocoto` directory there.

However, I can modify the code to copy `prep_stabias.sh` separately for  different `YAML_GEN_METHOD` if this really bothers you.

Reply:
What about we  put the prep_satbias.sh to workflow/ush?

Reply:
@guoqing-noaa I think that sounds reasonable. Although it should copied in the same way and is independent of `YAML_GEN_METHOD`, right?

Reply:
@delippi Yes, your are right.
Let me know if you still prefer copying it separately. Thanks!

Reply:
@guoqing-noaa, yes please copy it separately. I think my previous message might have been unclear. Here is pseudo code of what I meant. And hopefully this is more clear:
```
if YAML_GEN_METHOD == 1:
  copy qrocoto
if YAML_GEN_METHOD == 2:
  copy other rocoto tools

copy prep_satbias.sh expdir/.
```

I just meant that the copy of prep_satbias should be independent on the YAML_GEN_METHOD.